src/main/java/nl/tudelft/semgroup4/resources/Resources.java

```diff
@@ -1,120 +1,35 @@
 package nl.tudelft.semgroup4.resources;
 
-import java.awt.Font;
-import java.util.ArrayList;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
 
 import org.newdawn.slick.Animation;
 import org.newdawn.slick.Image;
 import org.newdawn.slick.Music;
 import org.newdawn.slick.SlickException;
 import org.newdawn.slick.Sound;
-import org.newdawn.slick.TrueTypeFont;
 
 /**
  * This class holds and loads the Slick2D resources.
+ * 
  * @author Justin
  */
 public final class Resources {
```

Comment:
PMD warns that this method should be a utility class. Adding a private constructor should fix that.

< interface